FrontEnd/src/components/MakeAPost/Page1.tsx

```diff
@@ -31,6 +36,13 @@ export const page1InitialStore: Page1Store = {
   placeName: '',
 };
 
+interface DropDownProps {
+  setStore: SetStore<Page1Store>;
```

Comment:
no need for `setStore` or `validations` since the `RoomDropdown` is defined inside of `Page1` and already has access to them.